provisioner/utils/smoke-tests.sh

```diff
@@ -67,6 +67,14 @@ ALL_SMOKE_TASKS=""
 for s in `echo $SMOKE_TESTS | sed -e 's#,# #g'`; do
   ALL_SMOKE_TASKS="$ALL_SMOKE_TASKS bigtop-tests:smoke-tests:$s:test"
 done
+
+# BIGTOP-4069 create the soft link of python for openEuler smoke-tests
+. /etc/os-release
+OS="$ID"
+if [[ $SMOKE_TESTS == *"phoenix"* && $OS == "openEuler" ]]; then
+    ln -s /usr/bin/python3 /usr/bin/python
+fi
+
```

Comment:
The test script should not modify system environment since we can not assume that the script is only used in disposable containers for tests.

In addition, this looks just hiding packaging issue. psql.py should be conditionally patched if [the package requires python3 in specific platforms](https://github.com/apache/bigtop/blob/master/bigtop-packages/src/rpm/phoenix/SPECS/phoenix.spec#L97-L101).

If all package can assume python3 on openEuler, the symlink should be created by [python.pp](https://github.com/apache/bigtop/blob/master/bigtop-deploy/puppet/manifests/python.pp).

Reply:
@iwasakims 
We modifed the code from your advice, please help check it, thanks.

Reply:
> The test script should not modify system environment since we can not assume that the script is only used in disposable containers for tests.
> 
> In addition, this looks just hiding packaging issue. psql.py should be conditionally patched if [the package requires python3 in specific platforms](https://github.com/apache/bigtop/blob/master/bigtop-packages/src/rpm/phoenix/SPECS/phoenix.spec#L97-L101).
> 
> If all package can assume python3 on openEuler, the symlink should be created by [python.pp](https://github.com/apache/bigtop/blob/master/bigtop-deploy/puppet/manifests/python.pp).

@iwasakims 
> If all package can assume python3 on openEuler, the symlink should be created by [python.pp]
We modifed the code, please help check it, thanks.